src/index.js

```diff
@@ -1,34 +1,87 @@
 import './style.css';
+import {
+  createTask,
+  saveItemToLocalStorage,
+  getItemFromLocalStorage,
+  deleteCompletedTasks,
+  deleteTask,
+} from './activity.js';
 
-const toDo = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Take a walk',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Go to the movie club with friends',
-    completed: false,
-    index: 2,
-  },
-];
-
-const displayToDoList = (items) => {
+const displayTasks = () => {
+  const tasks = getItemFromLocalStorage();
   const toDoList = document.getElementById('to-do-list');
-  items.forEach((toDo) => {
+  toDoList.innerHTML = '';
+  tasks.forEach((task, index) => {
     const toDoItem = document.createElement('li');
-    toDoItem.className = ('todo-item-list');
-    toDoItem.innerHTML = `<input type = "checkbox" class = "checkbox"${toDo.completed ? 'checked' : ''}></input>
-    <p>${toDo.description}</p>`;
+    toDoItem.className = ('to-do-item-list');
+
+    const anotherItem = document.createElement('div');
+    anotherItem.className = 'another-item';
+
+    const taskInput = document.createElement('input');
+    taskInput.className = 'checkbox';
+    taskInput.type = 'checkbox';
+    if (task.completeted) {
+      taskInput.setAttribute('checked', '');
+    }
+
+    taskInput.onclick = (e) => {
+      if (e.target.checked) {
+        tasks[index].completed = true;
+        e.target.parentNode.children[1].classList.add('strike-through');
+      } else {
+        tasks[index].completed = false;
+        e.target.parentNode.children[1].classList.remove('strike-through');
+      }
+      saveItemToLocalStorage(tasks);
+    };
+
+    anotherItem.appendChild(taskInput);
+
+    const taskDescription = document.createElement('p');
+    taskDescription.classList.add('description');
+    if (task.completed) {
+      taskDescription.classList.add('strike-through');
+    } else {
+      taskDescription.classList.remove('strike-through');
+    }
+    taskDescription.innerText = task.description;
+
+    anotherItem.appendChild(taskDescription);
+    toDoItem.appendChild(anotherItem);
+
+    const deleteBtn = document.createElement('button');
+    deleteBtn.className = 'delete-btn';
+    deleteBtn.innerHTML = ':';
+    deleteBtn.addEventListener('onclick', () => {
+      deleteTask();
+      displayTasks();
+    });
```

Comment:
- [ ] After implementing the UI changes from the previous reviewer, I noticed that you have not made any provisions for the delete button to show up when a user clicks on any to-do item so they can be deleted. Please watch this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) to get a better grasp of what is expected of you 👍